Update image-builder-troubleshoot.md (#173)
MicrosoftDocs/azure-compute-docs: articles/virtual-machines/linux/image-builder-troubleshoot.md

```

For more information about configuring permissions, see [Configure VM Image Builder permissions by using the Azure CLI](image-builder-permissions-cli.md) or [Configure VM Image Builder permissions by using PowerShell](image-builder-permissions-powershell.md).

### Not Authorized to Access resource

#### Error

```output
Not authorized to access the resource: <resource-not-able-to-access>. Please check the user assigned identity has the correct permissions. For more details, go to https://aka.ms/azvmimagebuilderts.
```

#### Cause

 The created [Managed Service Identities (MSI)](./image-builder-permissions-cli.md#create-a-user-assigned-managed-identity) assigned to the image template does not have all permissions to access the resource shared on the error message.
 
#### Solution

Confirm the managed identity from the target image builder template

```azurecli-interactive
az image builder identity show -g <template resource group> -n <template name> 
```

Review the role assignments for the identity:

```azurecli-interactive
az role assignment list --assignee <identity_client_id_or_principal_id>
``` 

> Assign the require role or if required create you role with the required permissions.

Reviewer: ```suggestion
Assign the required role, or if required create your role with the required permissions.
```
@kof-f Please let me know if I've understood this line correctly with my edits.
